build.gradle

```diff
@@ -29,6 +28,7 @@ buildscript {
 plugins {
     id "java"
     id "com.github.hierynomus.license" version "0.14.0"
+    id "net.ltgt.apt" version "0.19"
```

Comment:
need to upgrade the plugin to resolve the bug which occurs when building SW as `./gradlew build -x check --include-build /path/to/h2o`